test/e2e/images.bzl

```diff
@@ -66,7 +66,7 @@ def install():
         registry = "ghcr.io",
         repository = "kyverno/kyvernopre",
         tag = "v1.3.6",
-        digest = "sha256:e76fb71c59449bca1028724a88005652409b56efb90bbcdce56b0d083bda6568",
+        digest = "sha256:94fc7f204917a86dcdbc18977e843701854aa9f84c215adce36c26de2adf13df",
```

Comment:
crane digest ghcr.io/kyverno/kyvernopre:v1.3.6
sha256:e76fb71c59449bca1028724a88005652409b56efb90bbcdce56b0d083bda6568

Is this expected?

Reply:
> Is this expected?

It is; this is super confusing, and I should've added it to the PR description, sorry. Bazel wants the digest of the actual image; `crane digest ghcr.io/kyverno/kyvernopre:v1.3.6` gives the digest of the _manifest list_ rather than the image itself.

```console
$ crane manifest ghcr.io/kyverno/kyvernopre:v1.3.6
{
   "mediaType": "application/vnd.docker.distribution.manifest.list.v2+json",
   "schemaVersion": 2,
   "manifests": [
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "digest": "sha256:f1a85fb6a95ccc9770e668116e0252c7e7c42b6403f3451047e154b8367cb987",
         "size": 735,
         "platform": {
            "architecture": "arm64",
            "os": "linux"
         }
      },
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "digest": "sha256:94fc7f204917a86dcdbc18977e843701854aa9f84c215adce36c26de2adf13df",
         "size": 736,
         "platform": {
            "architecture": "amd64",
            "os": "linux"
         }
      }
   ]
}
```

> (Should the tag and the sha256 be updated at the same time?)

We want to keep the tag the same for now (we'll update it in #4507); the `v1.3.6` tag was changed upstream to point to a different digest, which is why we have to update the digest here. This PR reflects the changed upstream.

Reply:
👍  Makes sense, so you're updating the SHA256 to point to the linux/amd64 digest for v1.3.6.